src/null-safety/index.md

```diff
@@ -3,13 +3,36 @@ title: Sound null safety
 description: Information about Dart's null safety feature
 ---
 
-The Dart language now supports sound null safety!
+The Dart language comes with sound null safety.
+
+Null safety prevents errors that result from unintentionel access
+of variables set to `null`.
+For example, if a method expects an integer but receives `null`,
+your app causes a runtime error. This type of error, a null dereference error,
+can be difficult to debug.
+
+With sound null safety variables are 'non-nullable' by default.
+You can specify that the type of a variable is nullable,
+and only then can they contain a `null`.
+Sound null safety changes potential **runtime errors**
+into **edit-time** analysis errors.
+This allows you to fix these errors before deploying your app.
+
+{{site.alert.warn}}
+In Dart 2.x SDKs, you can enable or disable sound null safety 
+through configuration of the project SDK constraint.
+To learn more, see [Enabling/disabling null safety](#enable-null-safety).
+
+Dart 3 -- tentatively planned for release by mid 2023 --
```

Comment:
**nit:** [Google style does not use spaces before or after em dashes](https://developers.google.com/style/dashes?hl=en#em-dashes).
```suggestion
Dart 3--planned for a mid-2023 release--
```

**issue:** Plans are tentative. The adverb adds no value to the sentence. I would remove it.